<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Door">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">\background-color: rgb(224, 198, 255);
color: rgb(0, 0, 0);
font: 75 10pt &quot;Norasi&quot;;</string>
    </property>
    <property name="text">
     <string>OPEN DOOR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(224, 198, 255);
color: rgb(0, 0, 0);
font: 75 10pt &quot;Norasi&quot;;</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="push_reset">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(224, 198, 255);
color: rgb(0, 0, 0);
font: 75 10pt &quot;Norasi&quot;;
</string>
    </property>
    <property name="text">
     <string>RESET TIME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plus">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(224, 198, 255);
color: rgb(0, 0, 0);
font: 75 10pt &quot;Norasi&quot;;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="minus">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(224, 198, 255);
color: rgb(0, 0, 0);
font: 75 10pt &quot;Norasi&quot;;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);
color: rgb(0, 255, 255);
font: 75 16pt &quot;Norasi&quot;;
color: rgb(0, 255, 255);</string>
    </property>
    <property name="text">
     <string>LED LIGHT LIFE TIME</string>
    </property>
   </widget>
   <widget class="QLabel" name="Date_time">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 127);
font: 75 12pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="date">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 75 12pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="f">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_doorstatus">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);
color: rgb(0, 255, 255);
font: 75 16pt &quot;Norasi&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Door_close">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>100</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(224, 198, 255);
color: rgb(0, 0, 0);
font: 75 10pt &quot;Norasi&quot;;</string>
    </property>
    <property name="text">
     <string>CLOSE DOOR</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="f_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);
font: 14pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="f_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>31</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);
font: 14pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>MACHINE STATUS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Norasi&quot;;
color: rgb(85, 255, 0);</string>
    </property>
    <property name="text">
     <string>Remaining</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_sec">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_min">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>280</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_hr">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Norasi&quot;;
color: rgb(85, 255, 0);</string>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Norasi&quot;;
color: rgb(85, 255, 0);</string>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_skip">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>160</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">\background-color: rgb(224, 198, 255);
color: rgb(0, 0, 0);
font: 75 10pt &quot;Norasi&quot;;</string>
    </property>
    <property name="text">
     <string>SKIP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_preload">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(224, 198, 255);
color: rgb(0, 0, 0);
font: 75 10pt &quot;Norasi&quot;;
</string>
    </property>
    <property name="text">
     <string>PRE-LOAD</string>
    </property>
   </widget>
   <widget class="QLabel" name="mstatus">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>46</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Norasi&quot;;
color: rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sensor">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>HIt Sensor</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOBJECTS">
    <property name="title">
     <string>OBJECTS </string>
    </property>
    <addaction name="actionWOOD"/>
    <addaction name="actionSTEEL"/>
    <addaction name="actionPLASTIC"/>
    <addaction name="actionGLASS_2"/>
   </widget>
   <addaction name="menuOBJECTS"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionWOOD">
   <property name="text">
    <string>WOOD</string>
   </property>
  </action>
  <action name="actionSTEEL">
   <property name="text">
    <string>STEEL</string>
   </property>
  </action>
  <action name="actionPLASTIC">
   <property name="text">
    <string>PLASTIC</string>
   </property>
  </action>
  <action name="actionGLASS_2">
   <property name="text">
    <string>GLASS</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
